### PR TITLE
Jenkins test branch - don't merge

### PR DESCRIPTION
**Pull request by jcellary**
This is just a test
@WikiaSparrow
test

### CODE REVIEW

**Comment by jcellary** [comment]
This is a comment @WikiaSparrow

File: README.rst [hunk 1]
```diff
@@ -1,4 +1,5 @@
 Sparrow
+just a test
 =======
```
Comment (jcellary) [inline README.rst]: This is a line comment @WikiaSparrow

**Comment by jcellary** [comment]
PR Comment @WikiaSparrow

**Comment by wladekb** [comment]
This is a test comment!

**Comment by wladekb** [comment]
This is a second test comment!

**Pull request closed without merging**